web-console/knikubevirt/snapshots/snapshots.md

```diff
@@ -0,0 +1,77 @@
+# Virtual Machines Snapshots
+
+Snapshots allow users to create a copy of the VM disks and memory, mainly for restore and backup purposes. The snapshots are taken live or offline depending on if the VM is running or not. 
+This document describes the snapshot design flow.  
+
+
+
+
+## From VM List 
+
+![Offline VM list - snapshot action from a single VM kebab](img/1-00.jpg)
```

Comment:
Probably just an oversight, but list page headers in OpenShift don't have object type badges next to them, only object detail pages